credible/cli-reference.mdx

```diff
@@ -0,0 +1,259 @@
+  status  Check current authentication status
+  help    Print this message or the help of the given subcommand(s)
+
+Options:
+  -u, --auth-url <AUTH_URL>  Base URL for authentication service [env: PCL_AUTH_URL=] [default: https://dapp.internal.phylax.systems]
+  -h, --help                 Print help
+```
+
+#### `auth login`
+
+Login to `pcl` using your wallet. Opens a browser window for wallet authentication.
+
+#### `auth logout`
+
+Logout from `pcl` and clear local authentication token.
+
+#### `auth status`
+
+Check current authentication status.
+
+### `test`
+
+Run tests for your assertions.
+
+```bash
+Run tests
+
+Usage: pcl test [OPTIONS] [PATH]
+
+Options:
+  -h, --help  Print help (see more with '--help')
+
+Common options:
+  -v, --verbosity...            Verbosity level of the log messages. Pass multiple times to increase verbosity (e.g. -v, -vv, -vvv)
+  -q, --quiet                   Do not print log messages
+      --match-path <GLOB>       Only run tests in source files matching the specified glob pattern
+      --match-test <REGEX>      Only run test functions matching the specified regex pattern
+      --match-contract <REGEX>  Only run tests in contracts matching the specified regex pattern
+```
+
+<Note>
+`pcl test` supports all Forge test options. See the [Forge test documentation](https://book.getfoundry.sh/reference/forge/forge-test) for additional options.
+</Note>
+
+### `build`
+
+Build contracts using [`phorge`](/credible/glossary#phorge).
+
+```bash
+Build contracts using `phorge`
+
+Usage: pcl build [OPTIONS]
+
+Options:
+      --root <ROOT>  Root directory of the project
+  -h, --help         Print help
+```
+
+<Note>
+`pcl build` will be run automatically if the project hasn't been built yet when using `pcl store`.
+</Note>
+
+### `store`
+
+Submit [assertion bytecode](/credible/glossary#assertion-bytecode) and source code to the [Assertion DA](/credible/glossary#assertion-da-data-availability).
+
+```bash
+Submit the Assertion bytecode and source code to be stored by the Assertion DA of the Credible Layer
+
+Usage: pcl store [OPTIONS] <ASSERTION_CONTRACT> [CONSTRUCTOR_ARGS]...
+
+Arguments:
+  <ASSERTION_CONTRACT>   Name of the assertion contract to build and flatten
+  [CONSTRUCTOR_ARGS]...  Constructor arguments for the assertion contract.
+                                Format: <ARG0> <ARG1> <ARG2>
+
+Options:
+  -u, --da-url <DA_URL>   URL of the assertion-DA server [env: PCL_DA_URL=]
+                          [default: https://da.internal.phylax.systems]
+      --root <ROOT>       Root directory of the project
+  -h, --help              Print help (see more with '--help')
+```
+
+**Examples:**
+
+```bash
+# Store assertion contract without constructor arguments
+pcl store MyAssertionContract
+
+# Store assertion with constructor arguments
+pcl store MyAssertion 0xADDRESS_OF_SMART_CONTRACT
+
+# Store assertion with multiple constructor arguments
+pcl store MyAssertion 0xADDRESS arg1 arg2
+```
+
+<Note>
```

Comment:
I think that this comment requires a lot of context from the reader and it's confusing